protos/protos/public/registry/RegistryService.proto

```diff
@@ -32,9 +32,8 @@ message RegisteredModel {
     // Visibility of Registered Models
     common.VisibilityEnum.Visibility visibility = 7;
 
-    // id of the workspace this belongs to
-    //not storing workspace_type uac should support https://github.com/VertaAI/protos-all/pull/19/files
-    string workspace_id = 8;
+    // Legacy id of the workspace this belongs to
+    string legacy_workspace_id = 8;
```

Comment:
Changing names breaks backwards compatibility with the client. Please check with Michael if the field is used.